src/wifi.rs

```diff
@@ -799,6 +799,49 @@ impl<'d> WifiDriver<'d> {
         })
     }
 
+    /// Get information of AP which the ESP32 station is associated with.
+    /// Usefull to get the current signal strength of the AP, in a simple way.
```

Comment:
```suggestion
    /// Useful to get the current signal strength of the AP.
```

Reply:
fixed in: f2bc5dd9b10a6710997a82cbe6bcf8d1297e4897